cypress/support/step_definitions/dictionary/addConceptsToDictionary/addConceptsToDictionary.ts

```diff
@@ -0,0 +1,170 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  After,
+  Before,
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+When('the user selects "Pick concepts"', () => {
+  cy.visit(
+    `/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`
+  );
+});
+
+Then('the user selects "Import existing concept"', () => {
+  cy.findByText("Import existing concept").click();
+});
+
+Then('the user should be on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+Given('the user is on the "Import existing concept" page', () => {
+  cy.visit(
+    `/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`
+  );
+  cy.url().should(
+    "contain",
+    `/orgs/CIEL/sources/CIEL/concepts/?addToDictionary=/users/${getUser()}/collections/${getDictionaryId()}/`
+  );
+});
+
+Then('the current source should be "CIEL"', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+When('the user clicks on the row for "Serum"', () => {
+  cy.findByText("Serum")
+    .parent()
+    .next()
+    .click();
+});
+
+When('the user clicks the "Add Serum to dictionary" button', () => {
```

Comment:
```suggestion
When('the user clicks on the "Add Serum to dictionary" button', () => {
```

Reply:
@jayasanka-sack and @ibacher  it has be a tough journey but am glad am remaining with 1 error though the error on the PR is different from the one on the terminal.